packages/recipe/src/components/EzAlert/EzAlert.tsx

```diff
@@ -15,7 +15,7 @@ const icons = {
 type Props = {
   arrow?: 'top' | 'bottom';
   headline: string;
-  tagline?: string | Element;
+  tagline?: string | Element | React.ReactNode;
```

Comment:
Lets get rid of `Element` and replace it with `ReactNode` here (and import it at the top).
```suggestion
  tagline?: string | ReactNode;
```

Reply:
Part of the reason why I did React.ReactNode here was that the import without direct usage was considered to be a linting error. I'm not sure why the linter wasn't picking up the type definition as "usage", but this was my way of skirting around having to define that import with the lint ignore line.

Reply:
Strange! Seems fine on my end, but I'm not worried about it. 👍